packages/venia-ui/lib/components/Checkout/flow.js

```diff
@@ -72,7 +73,7 @@ const Flow = props => {
                 <div className={classes.footer}>
                     <CheckoutButton
                         disabled={checkoutDisabled}
-                        onClick={handleBeginCheckout}
+                        onClick={handleGoToCart}
```

Comment:
If a developer is upgrading from 6.0.0 to 7.0.0 and doesn't want to use the new cart/checkout experience they can simply set this back to `handleBeginCheckout`.